xml/schema/Activity/Activity.xml

```diff
@@ -266,6 +266,7 @@
     <name>priority_id</name>
     <type>int unsigned</type>
     <title>Priority</title>
+    <import>true</import>
```

Comment:
@alifrumin can you also add this property in [Activity DAO](https://github.com/civicrm/civicrm-core/blob/master/CRM/Activity/DAO/Activity.php#L439)  ?